<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="login_button">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>ΕΙΣΟΔΟΣ </string>
   </property>
  </widget>
  <widget class="QLabel" name="user_label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ΧΡΗΣΤΗΣ : </string>
   </property>
  </widget>
  <widget class="QLabel" name="password_label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ΚΩΔΙΚΟΣ : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password_lineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>120</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="quit_button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>ΕΞΟΔΟΣ</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="user_lineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="title_label">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ΑΛΛΕΡΓΙΟΛΟΓΙΚΟ ΠΡΟΓΡΑΜΜΑ</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="register_button">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>REGISTER</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
